FirebaseAppDistribution/Sources/FIRFADApiService.m

```diff
@@ -72,9 +75,15 @@ + (NSMutableURLRequest *)createHTTPRequest:(NSString *)method
   FIRFADInfoLog(@"Requesting releases for app id - %@", [[FIRApp defaultApp] options].googleAppID);
   [request setURL:[NSURL URLWithString:urlString]];
   [request setHTTPMethod:method];
-  [request setValue:authTokenResult.authToken forHTTPHeaderField:kInstallationAuthHeader];
-  [request setValue:[[FIRApp defaultApp] options].APIKey forHTTPHeaderField:kApiHeaderKey];
-  [request setValue:[NSBundle mainBundle].bundleIdentifier forHTTPHeaderField:kApiBundleKey];
+  [request setValue:authTokenResult.authToken forHTTPHeaderField:kFIRFADInstallationAuthHeader];
+  [request setValue:[[FIRApp defaultApp] options].APIKey forHTTPHeaderField:kFIRFADApiHeaderKey];
+  [request setValue:[NSBundle mainBundle].bundleIdentifier forHTTPHeaderField:kFIRFADApiBundleKey];
+  [request setValue:[FIRApp firebaseUserAgent] forHTTPHeaderField:kFIRFADFirebaseUserAgentKey];
+  dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_LOW, 0), ^{
+    [request setValue:@([FIRHeartbeatInfo heartbeatCodeForTag:kFIRFADHeartbeatTag]).stringValue
```

Comment:
As we figured out recently, the server side integration with platform logging pipeline should be implemented before the SDK release, because otherwise if the SDK reports the global heartbeat, it won't be reported by other SDKs and will be lost which will affect the stats for all SDKs.

Please let me know if you need assistance with it.